content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/differences.rst

```diff
@@ -2,46 +2,94 @@
 
-**Serial numbers** are identification numbers given to one product in
-particular, to allow to track the history of the item from reception to
-delivery and after-sales.
+Enable lots & serial numbers
+============================
 
-When to use
------------
+To track products using lots and serial numbers, the *Lots & Serial Numbers* feature must be
+enabled. Go to the :menuselection:`Inventory app --> Configuration --> Settings`, scroll down to the
+:guilabel:`Traceability` section, and click the box next to :guilabel:`Lots & Serial Numbers`. Then,
+click the :guilabel:`Save` button to save changes.
 
-**Lots** are interesting for products you receive in great quantity and for
-which a lot number can help in reportings, quality controls, or any
-other info. Lots will help identify a number of pieces having for
-instance a production fault. It can be useful for a batch production of
-clothes or food.
+.. image:: differences/differences-enabled-setting.png
+   :align: center
+   :alt: Enabled lots and serial numbers feature in inventory settings.
 
-**Serial numbers** are interesting for items that could require after-sales
-service, such as smartphones, laptops, fridges, and any electronic
-devices. You could use the manufacturer's serial number or your own,
-depending on the way you manage these products
+When to use lots
+================
 
-When not to use
----------------
+Lots are useful for products that are manufactured or received in large quantities (such as clothes
+or food), and can be used to trace a product back to a group. This is especially useful when
+managing product recalls or expiration dates.
 
-Storing consumable products such as kitchen roll, toilet paper, pens and
-paper blocks in lots would make no sense at all, as there are very few
-chances that you can return them for production fault.
+.. example::
+   .. image:: differences/differences-lot.png
+      :align: center
+      :alt: Created lot with quantity of products in it.
 
-On the other hand, giving a serial number to every product is a
-time-consuming task that will have a purpose only in the case of items
-that have a warranty and/or after-sales services. Putting a serial
-number on bread, for instance, makes no sense at all.
+Manufacturers assign lot numbers to groups of products that have common properties; this can lead to
+multiple goods sharing the same lot number. This helps to identify a number of products in a single
+group, and allows for end-to-end traceability of these products through each step in their
+life cycles.
 
-.. seealso::
-    * :doc:`serial_numbers`
-    * :doc:`lots`
+When to use serial numbers
+==========================
+
+The goal of assigning serial numbers to individual products is to make sure that every item's
+history is identifiable when it travels through the supply chain. This can be especially useful for
+manufacturers that provide after-sales services to products that they sell and deliver.
```

Comment:
```suggestion
The goal of assigning serial numbers to individual products is to make sure every item's history is
identifiable when it travels through the supply chain. This can be especially useful for
manufacturers that provide after-sales services related to products they sell and deliver.
```